### PR TITLE
subset baseline samples

### PR DESCRIPTION
**Pull request by YagmurSimsekk**

### CODE REVIEW

**Comment by YagmurSimsekk** [comment]
Third example in #22 is not possible with HITChip data since it doesn't have any NA time points for any subject. Should I use a different data set for this issue?

**Comment by antagomir** [comment]
Write there 

```
# Add some NAs in the time point data to demonstrate functionality with missing data
nainds <- sample(seq_len(ncol(hitchip1006)), 50)
colData(hitchip1006)$time[nainds] <- NA
```

File: vignettes/miaTime.Rmd [hunk 1]
```diff
@@ -215,7 +230,7 @@ BiocManager::install("mia")
 ```
 
 
-```{r, eval=FALSE}
+```{r eval=FALSE, include=FALSE}
```
Comment (antagomir) [inline vignettes/miaTime.Rmd:233]: could we have eval=TRUE here and elsewhere? would be better for constant testing, if possible

Comment (YagmurSimsekk) [reply vignettes/miaTime.Rmd]: That was because I thought it would give error due to mia dependency. It is removed now, ready to be tested.

**Comment by antagomir** [comment]
Seems to go through nicely!

**Pull request merged**